config/purposes/lcm_triomics.wip.yml

```diff
@@ -0,0 +1,91 @@
+# EMSeq plate purposes
+---
+LCMT Lysate:
+  :asset_type: plate
+  :stock_plate: true
+  :cherrypickable_target: false
+  :input_plate: true
+  :presenter_class: Presenters::StockPlatePresenter
+  :creator_class: LabwareCreators::Uncreatable
+  :size: 96
+LCMT DNA Frag:
+  :asset_type: plate
+  :stock_plate: false
+  :cherrypickable_target: false
+  :input_plate: false
+  :presenter_class: Presenters::PermissivePresenter
+  :size: 96
+LCMT DNA End Prep:
+  :asset_type: plate
+  :stock_plate: false
+  :cherrypickable_target: false
+  :input_plate: false
+  :presenter_class: Presenters::PermissivePresenter
+  :size: 96
+LCMT DNA Adp Lig:
+  :asset_type: plate
+  :stock_plate: false
+  :cherrypickable_target: false
+  :input_plate: false
+  :presenter_class: Presenters::PermissivePresenter
+  :size: 96
+LCMT EM TET2 Ox:
+  :asset_type: plate
+  :stock_plate: false
+  :cherrypickable_target: false
+  :input_plate: false
+  :presenter_class: Presenters::PermissivePresenter
```

Comment:
add size

Reply:
and ones below

Reply:
default is 96 I assume so could just not set size on ones above, but nice to be clear I think?

Reply:
I have now added the size 96 to the plate purposes explicitly.